native/acter/src/api/password_reset.rs

```diff
@@ -0,0 +1,100 @@
+use acter_core::events::password_reset::{
+    PasswordResetContent, PasswordResetViaEmail, PasswordResetViaPhone,
+};
+use anyhow::{Context, Result};
+use std::ops::Deref;
+
+use super::{account::Account, RUNTIME};
+
+#[derive(Clone)]
+pub struct PasswordReset {
+    inner: PasswordResetContent,
+}
+
+impl Deref for PasswordReset {
+    type Target = PasswordResetContent;
+    fn deref(&self) -> &Self::Target {
+        &self.inner
+    }
+}
+
+impl PasswordReset {
+    fn via_email(&self) -> Option<PasswordResetViaEmail> {
+        self.inner.via_email()
+    }
+
+    fn via_phone(&self) -> Option<PasswordResetViaPhone> {
+        self.inner.via_phone()
+    }
+}
+
+impl Account {
+    pub async fn set_password_reset_via_email(
+        &self,
+        submit_url: Option<String>,
```

Comment:
what is submit_url? That doesn't really make any sense ...

Reply:
`submit_url` is used in `submit_token_from_email()`.